msdm/domains/gridworld/mdp.py

```diff
@@ -65,10 +65,14 @@ def __init__(self,
             feature_rewards = {'g': 0}
         self._featureRewards = feature_rewards
         self.step_cost = step_cost
-        self.termination_prob = termination_prob #basically discount rate
+        self.discount_rate = discount_rate
```

Comment:
If this isn't being referenced in `next_state_dist`, should we remove?

Reply:
resolved?

Reply:
yes!